packages/react-liveness/src/components/FaceLivenessDetector/service/utils/liveness.ts

```diff
@@ -12,10 +12,8 @@ import {
 import { ColorSequence, SequenceColorValue } from './ColorSequenceDisplay';
 import {
   FACE_HEIGHT_WEIGHT,
+  OVAL_HEIGHT_WIDTH_RATIO,
   PUPIL_DISTANCE_WEIGHT,
-  FACE_DISTANCE_THRESHOLD,
```

Comment:
So are these values now coming from the config instead?

Reply:
yep!